dof/Cargo.toml

```diff
@@ -1,8 +1,10 @@
 [package]
 name = "dof"
 version = "0.1.5"
-authors = ["Benjamin Naecker <ben@oxidecomputer.com>",
-           "Adam H. Leventhal <ahl@oxidecomputer.com>"]
+authors = [
+  "Benjamin Naecker <ben@oxidecomputer.com>",
+  "Adam H. Leventhal <ahl@oxidecomputer.com>",
+]
```

Comment:
delete

Reply:
Delete the authors or delete the change? I think it's only changed due to autoformatting.